spec/services/remove_item_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'RemoveItemService' do
+  subject(:remove_item) { CartServices::RemoveItem.new }
+
+  let!(:product) { create :product }
+  let!(:item) { create :cart_item, :with_cart }
+  let!(:unauthorized_user) { create :user }
```

Comment:
You can split it two contexts'
1.` context 'when user is authorized'`
`it 'is allowed to removes an item from the cart'`
2. `context 'when user is unauthorized'`
`it 'is not allowed to removes an item from the cart'`